src/object_properties.h

```diff
@@ -54,6 +54,7 @@ struct ObjectProperties
 	std::string infotext;
 	//! For dropped items, this contains item information.
 	std::string wield_item;
+	bool no_static_save = false;
```

Comment:
please use positive meaning. static_save = true and invert logic (except for lua ?)